src/support_sphere/test/unit_tests/auth_service_test.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:support_sphere/data/services/auth_service.dart';
+
+
+void main() {
+
+  late AuthService authService;
+
+  setUp(() {
+    authService = AuthService();
+  });
+
+  group('AuthService', () {
```

Comment:
What is `group` vs `setUp`? Could you provide a little bit of description for these two? Thanks!

Reply:
Updated